scripts/bounty-awaiting-approval-slack-ping.js

```diff
@@ -0,0 +1,90 @@
+// Description:
+//   Script that listens for issues with the label 'bounty-awaiting-approval'
+//   and notifies the collaborators on Slack.
+//
+// Dependencies:
+//   github: "^13.1.0"
+//   probot-config: "^0.1.0"
+//   probot-slack-status: "^0.2.2"
+//
+// Author:
+//   Max Tyrrell (ImFeelingDucky/mac/yung_mac)
+
+const defaultConfig = require('../lib/config')
+const Slack = require('probot-slack-status')
+
+module.exports = (robot) => {
+  robot.log('Connected to bounty-awaiting-approval-slack-ping')
+
+  Slack(robot, (slack) => {
+    robot.log.trace('Connected to Slack')
+
+    checkForNewBounties(robot, slack)
+  })
+}
+
+function checkForNewBounties (robot, slackClient) {
+  robot.on('issues.labeled', async context => {
+    // Make sure we don't listen to our own messages
+    if (context.isBot) return null
+
+    await notifyCollaborators(context, robot, slackClient)
+  })
+}
+
+async function notifyCollaborators (context, robot, slackClient) {
+  const { github, payload } = context
+  const ownerName = payload.repository.owner.login
+  const repoName = payload.repository.name
+  const config = defaultConfig(robot, '.github/github-bot.yml')
+
+  if (!config['bounty-awaiting-approval-slack-ping']) return null
+
+  const watchedLabelName = config['bounty-awaiting-approval-slack-ping']['label-name']
+  if (payload.label.name !== watchedLabelName) {
+    robot.log.debug(`bountyAwaitingApprovalSlackPing - ${payload.label.name} doesn't match watched ${watchedLabelName} label. Ignoring`)
+    return null
+  }
+
+  robot.log(`bountyAwaitingApprovalSlackPing - issue #${payload.issue.number} on ${ownerName}/${repoName} was labeled as a bounty awaiting approval. Pinging slack...`)
+
+  const slackCollaborators = getSlackCollaborators(ownerName, repoName, github, robot)
+
+  // Don't actually send Slack messages if we're just doing a dry-run
+  if (process.env.DRY_RUN) {
```

Comment:
`slackHelper` already takes care of this logic, so you can get rid of this block